ReactiveXComponent/Configuration/XCConfiguration.cs

```diff
@@ -7,37 +8,106 @@ public class XCConfiguration : IXCConfiguration
     {
         private readonly XCApiConfigParser _parser;
 
-        public XCConfiguration(Parser.XCApiConfigParser parser)
+        public XCConfiguration(XCApiConfigParser parser)
```

Comment:
you should provide an interface of your parser (instead of implementation)
easier to test you configuration class